app/Helpers/CheckRateableHelper.php

```diff
@@ -2,8 +2,8 @@
 
 namespace App\Helpers;
 
-class CheckRateable{
-    public static function checkRateable($content_type, $content_id, $user_id){
+class CheckRateable {
+    public static function checkRateable($content_type, $content_id, $user_id) {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->